README.md

```diff
@@ -295,6 +295,19 @@ gives:
 +-----------+------+------------+-----------------+
 ```
 
+##### Other alignment
+
+You can also change the alignment of the headers of the columns using the `header_align`
+attribute in the same way as the `align` attribute.
+
+```python
+x.header_align["City name"] = "l"
+print(x)
+```
+
+The valign attribute works in the same way as the other align attributes for controlling
```

Comment:
```suggestion
The `valign` attribute works in the same way as the other align attributes for controlling
```

Reply:
And shall we mention the possible values for `valign`?

Reply:
Great idea! If there are any other changes you want me to add please let me know.